spinnaker-jenkins-to-vmss/scripts/set_azure_spinnaker.sh

```diff
@@ -109,9 +109,12 @@ sudo printf "rosco.yml file has been updated\n"
 sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EB9B1D8886F44E2A
 sudo printf "apt-key done\n" 
 
+# Enable Azure provider in /etc/default/spinnaker
+sudo sed -i '/SPINNAKER_AZURE_ENABLED=false/s/.*/SPINNAKER_AZURE_ENABLED=true/' /etc/default/spinnaker
```

Comment:
Per offline discussion I will merge this commit. This shouldn't "hurt" anything, but we are "enabling" Azure in two other places (line 60 when we call InstallSpinnaker.sh and line 73 where we make the sed call for "${SPINNAKER_AZURE_ENABLED:true}")

That means we do it in three places now and we should remove the two that don't work or are redundant